modules/administration/pages/image-management.adoc

```diff
@@ -17,7 +17,7 @@ The workflow looks like this:
 {productname} supports two distinct build types: dockerfile, and the Kiwi image system.
 
 The Kiwi build type is used to build system, virtual, and other images.
-The image store for the Kiwi build type is pre-defined as a filesystem directory at [path]``/srv/www/os-images`` on the server.
+The image store for the Kiwi build type is pre-defined as a file system directory at [path]``/srv/www/os-images`` on the server.
```

Comment:
this probably needs to be done suite-wide :woman_facepalming:

Reply:
I can use sed and grep to find the instances of filesystem, and then a script to find and replace them in each file.  Seems like something I should check out an issue with a plan of how I can write something to find and replace the use in text.

Reply:
Sure, go right ahead.